QUARKUS-5666.md

```diff
@@ -10,7 +10,8 @@
 ## Scope of the testing
 The goal of this testing is to ensure that this new feature introduced into Quarkus TLS Registry can successfully handle encrypted PKCS#8 (PEM) Keys.
 The tests validate both server and client side functionalities when using encrypted PEM certificates.
-Additionally, the tests should verify that is compatible with FIPS-enable environment.
+On the other hand, the tests won't be tested with FIPS-enable environment, it's currently not supported due to some limitations
+FIPS-compliant providers, which do not support the necessary encryption/decryption algorithms for encrypted PEM keys—resulting in null algorithm parameters and decryption failures.
```

Comment:
That seems like ultimate user experience. NPE and no documentation of this? Maybe something should be done about this?

Reply:
What I am trying to say:

- there will always be some dummies like me who can't lookup that AES-128-CBC is not approved for FIPS
- if we only support encrypted PEM in FIPS-enabled environment with fips incompatible algorithm, I think it should be NPE, but something clearer. what is actual logged error message?

TP says that you ensure good user experience. I think some docs note or ex. message improvement would be nice.